app/models/patient.rb

```diff
@@ -641,7 +641,11 @@ def report_eligibility
     # Can't send messages to monitorees that are on the closed line list and have no active dependents.
     if !monitoring && active_dependents.empty?
       eligible = false
-      messages << { message: 'Monitoree is not currently being monitored and has no actively monitored household members', datetime: closed_at }
+
+      # If this person has dependents (is a HoH)
+      is_hoh = !dependents_exclude_self.empty?
```

Comment:
This reads a little weird, what about `is_hoh = dependents_exclude_self.exists?`? `exists?` is also the the best option when you've still got something that is an `ActiveRecord::AssociationRelation`

Reply:
Agreed, changed.